app/models/student.rb

```diff
@@ -0,0 +1,8 @@
+class Student < ApplicationRecord
+  serialize :food_allergies, Array
+
```

Comment:
needs a `belongs_to :teacher` association.

Also add a `has_many :students` association on the teacher model.